vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -1,16 +1,13 @@
 # How to use layouts
 
-This guide shows you how to use the [`Layout`][vizro.models.Layout] to arrange charts/components on the screen and customize the grid specifications.
+The [`Page`][vizro.models.Page] model accepts a `layout` argument so that you can supply it with a custom grid. This guide shows how to use [`Layout`][vizro.models.Layout] to arrange charts and components on the screen so you can customize the grid specifications.
```

Comment:
Feel free to rephrase, but I think the order is a bit confusing here. What the user needs to understand:
1. Use layout to arrange charts and components as you like
2. Use the grid argument of the layout to customise the arrangement

```suggestion
The [`Page`][vizro.models.Page] model accepts a `layout` argument that allows for custom arrangement of charts and components on the screen. This guide shows how to customize the grid sepcifications in the [`Layout`][vizro.models.Layout].
```